src/app/accounts/create-account.ts

```diff
@@ -50,6 +50,7 @@ const setupAccount = async ({
   // FIXME: to remove when Casbin is been introduced
   const role = getTestAccounts().find(({ phone }) => phone === phoneNumberValid)?.role
   account.role = role
+  account.contactEnabled = !account.role || account.role === "user"
```

Comment:
I'm not sure this is very explicit on the intent: `!account.role`

Reply:
Based on [this comment](https://github.com/GaloyMoney/galoy/pull/1696/files#r979055348) should `editor` be included here too?

Reply:
fixed https://github.com/GaloyMoney/galoy/pull/1696/commits/ab3bd473794af580a5739e38e2862bf15a03b806